python/lsst/cp/verify/verifyBias.py

```diff
@@ -129,7 +131,7 @@ def verify(self, exposure, statisticsDict):
             # measured on the bulk of the image as the overscan
```

Comment:
I can't comment into the block above but this should say that the nominal detector read noise is in electron, and if you update the field this should be `READ_NOISEA_ADU {ampName}`.